app/views/patients/_patient_information.html.erb

```diff
@@ -111,6 +111,8 @@
                               checked: patient.special_circumstances.include?('Prison'), namespace: 'Prison' }, 'Prison', '' %>
             </div>
           <% end %>
+
+          <%= f.check_box :audited, label: 'Has patient fulfillment been audited?' %>
```

Comment:
should we move this to the fulfillment partial? I'm not totally sure we need to surface this info for non-data volunteers and admins, but interested in your thoughts.

Reply:
ps let's make sure to add this to the strong params in `patients_controller`